task_6.py

```diff
@@ -0,0 +1,4 @@
+names = ['Dean', 'Ally', 'Dean', 'Jane']
+names.pop(0)
```

Comment:
how do you know that nonunique value is at index 0? 
your code should work with any data.

Reply:
Where is fix?